.gitignore

```diff
@@ -1,3 +1,7 @@
 vendor
 .php_cs.cache
 composer.lock
+composer.phar
+clover.xml
+report
+result.txt
```

Comment:
Ok, I check this [link](https://github.com/Th3Mouk/YahooWeatherAPI/blob/master/.gitignore) to remove some additional lines.